### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by b4n**
Miscellaneous small fixes, mostly reported by GCC

### CODE REVIEW

**Comment by b4n** [comment]
> What kind of gcc option did you use?

`-g -Wall -W -O2 -Wunused -Wunreachable-code -Wformat=2 -Wundef -Wshadow -Wpointer-arith -Wstrict-prototypes -Wmissing-prototypes -Wmissing-declarations -Wmissing-noreturn -Wmissing-format-attribute -Wredundant-decls -Wnested-externs -Werror-implicit-function-declaration -fshow-column -Wno-unused-parameter -Wno-sign-compare -Wno-missing-field-initializers`, and `-Wwrite-strings` but this one emits a little too much noise in kind definitions (because `name` and `description` fields are not constants).

File: dsl/es-lang-c-stdc99.c [hunk 1]
```diff
@@ -688,7 +688,7 @@ es_symbol_p    (const EsObject*   object)
   return es_object_type_p(object, ES_TYPE_SYMBOL);
 }
 
-const char*
+static const char*
 es_symbol_name  (const EsObject*   object)
 {
```
Comment (masatake) [inline dsl/es-lang-c-stdc99.c]: Could you keep this "extern" ?
I found this function should be named es_symbol_get. es_symbol_get is declared in es-lang-c-stdc99.h.

File: dsl/qualifier.c [hunk 2]
```diff
@@ -351,10 +351,10 @@ static EsObject* builtin_eq  (EsObject *args, tagEntry *entry)
...
 }
 
-DEFINE_OP_WITH_CHECK(lt, es_number_get (a) < es_number_get (b), es_number_p,  NUMBER_REQUIRED, es_symbol_intern ("<"));
-DEFINE_OP_WITH_CHECK(gt, es_number_get (a) > es_number_get (b), es_number_p,  NUMBER_REQUIRED, es_symbol_intern (">"));
-DEFINE_OP_WITH_CHECK(le, es_number_get (a) <= es_number_get (b), es_number_p, NUMBER_REQUIRED, es_symbol_intern ("<="));
-DEFINE_OP_WITH_CHECK(ge, es_number_get (a) >= es_number_get (b), es_number_p, NUMBER_REQUIRED, es_symbol_intern (">="));
+DEFINE_OP_WITH_CHECK(lt, es_number_get (a) < es_number_get (b), es_number_p,  NUMBER_REQUIRED, es_symbol_intern ("<"))
+DEFINE_OP_WITH_CHECK(gt, es_number_get (a) > es_number_get (b), es_number_p,  NUMBER_REQUIRED, es_symbol_intern (">"))
+DEFINE_OP_WITH_CHECK(le, es_number_get (a) <= es_number_get (b), es_number_p, NUMBER_REQUIRED, es_symbol_intern ("<="))
+DEFINE_OP_WITH_CHECK(ge, es_number_get (a) >= es_number_get (b), es_number_p, NUMBER_REQUIRED, es_symbol_intern (">="))
```
Comment (masatake) [inline dsl/qualifier.c]: I would like to keep the last `;`.  It helps ctags to find a function.
`builtin_prefix` is not captured if we remove `;`.

File: dsl/es-lang-c-stdc99.c (hunk 1 above)
Comment (b4n) [reply dsl/es-lang-c-stdc99.c]: Fixed, and renamed the function in another commit

File: dsl/qualifier.c (hunk 2 above)
Comment (b4n) [reply dsl/qualifier.c]: OK, done

**Pull request merged**